A2/plugins/core.py

```diff
@@ -7,24 +10,37 @@
 class CorePlugin(Plugin):
     @Plugin.listen('Ready')
     def on_ready(self, ctx):
-        self.client.update_presence(Status.online, Game(type=GameType.listening, name='nothing because I don\'t work yet'))
+        game = Game(
+            type=GameType.listening,
+            name='nothing because I don\'t work yet')
+        self.client.update_presence(Status.online, game)
 
     @Plugin.command('reload')
     def reload_command(self, event):
+        """= reload =
+        Reloads all plugins except Core.
+        usage    :: $reload
+        aliases  :: None
+        category :: Core
+        """
         user_level = self.bot.get_level(event.author)
+
         if user_level < CommandLevels.OWNER:
-            event.msg.reply('**Only admins can do that**')
+            event.msg.reply('**Only admins can do that!**')
             return
 
         self.log.info('Reloading Plugins')
         reloaded_plugins = []
-        for name, value in self.bot.plugins.items():
-            if 'Core' in name:
+
+        for name, instance in self.bot.plugins.copy().items():
+            # Don't reload Core.
+            if instance == self:
                 continue
 
-            self.log.info('reloading {}'.format(name))
-            self.bot.reload_plugin(value.__class__)
-            self.log.info('reloaded {}'.format(name))
+            self.log.info('Reloading {}.'.format(name))
+            self.bot.reload_plugin(instance.__class__)
+            self.log.info('Reloaded {}.'.format(name))
             reloaded_plugins.append(name)
 
-        event.msg.reply('Plugins reloaded:\n{}'.format('\n'.join(reloaded_plugins)))
+        reloaded_plugins = '\n'.join(reloaded_plugins)
+        event.msg.reply('Plugins reloaded:\n```{}```'.format(reloaded_plugins))
```

Comment:
The first reloaded plugin does not show up in the message. The reason for this is because discord parses the first line of a code block as the language for syntax highlighting, and it is omitted from the actual message that is displayed.

Fix it by putting a `\n` right after the first triple backticks.